source/draw/Color.ooc

```diff
@@ -256,9 +248,23 @@ ColorBgra: cover {
2, 51968, 52224, 52480, 52736, 52992, 53248, 53504, 53760, 54016, 54272, 54528, 54784, 55040, 55296, 55552, 55808, 56064, 56320, 56576, 56832, 57088, 57344, 57600, 57856, 58112, 58368, 58624, 58880, 59136, 59392, 59648, 59904, 60160, 60416, 60672, 60928, 61184, 61440, 61696, 61952, 62208, 62464, 62720, 62976, 63232, 63488, 63744, 64000, 64256, 64512, 64768, 65024, 65280, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -45824, -45466, -45108, -44750, -44392, -44034, -43676, -43318, -42960, -42602, -42244, -41886, -41528, -41170, -40812, -40454, -40096, -39738, -39380, -39022, -38664, -38306, -37948, -37590, -37232, -36874, -36516, -36158, -35800, -35442, -35084, -34726, -34368, -34010, -33652, -33294, -32936, -32578, -32220, -31862, -31504, -31146, -30788, -30430, -30072, -29714, -29356, -28998, -28640, -28282, -27924, -27566, -27208, -26850, -26492, -26134, -25776, -25418, -25060, -24702, -24344, -23986, -23628, -23270, -22912, -22554, -22196, -21838, -21480, -21122, -20764, -20406, -20048, -19690, -19332, -18974, -18616, -18258, -17900, -17542, -17184, -16826, -16468, -16110, -15752, -15394, -15036, -14678, -14320, -13962, -13604, -13246, -12888, -12530, -12172, -11814, -11456, -11098, -10740, -10382, -10024, -9666, -9308, -8950, -8592, -8234, -7876, -7518, -7160, -6802, -6444, -6086, -5728, -5370, -5012, -4654, -4296, -3938, -3580, -3222, -2864, -2506, -2148, -1790, -1432, -1074, -716, -358, 0, 358, 716, 1074, 1432, 1790, 2148, 2506, 2864, 3222, 3580, 3938, 4296, 4654, 5012, 5370, 5728, 6086, 6444, 6802, 7160, 7518, 7876, 8234, 8592, 8950, 9308, 9666, 10024, 10382, 10740, 11098, 11456, 11814, 12172, 12530, 12888, 13246, 13604, 13962, 14320, 14678, 15036, 15394, 15752, 16110, 16468, 16826, 17184, 17542, 17900, 18258, 18616, 18974, 19332, 19690, 20048, 20406, 20764, 21122, 21480, 21838, 22196, 22554, 22912, 23270, 23628, 23986, 24344, 24702, 25060, 25418, 25776, 26134, 26492, 26850, 27208, 27566, 27924, 28282, 28640, 28998, 29356, 29714, 30072, 30430, 30788, 31146, 31504, 31862, 32220, 32578, 32936, 33294, 33652, 34010, 34368, 34726, 35084, 35442, 35800, 36158, 36516, 36874, 37232, 37590, 37948, 38306, 38664, 39022, 39380, 39738, 40096, 40454, 40812, 41170, 41528, 41886, 42244, 42602, 42960, 43318, 43676, 44034, 44392, 44750, 45108, 45466 ]
```

Comment:
To quote Star Trek: "There can be no justice so long as laws are absolute".

The first main rule says `Optimize for readability` and this seriously impacts that.